spec/rails_helper.rb

```diff
@@ -70,6 +70,10 @@
   config.include SidekiqTestHelpers
   config.include ElasticsearchHelpers, elasticsearch: true
 
+  config.before(:suite) do
```

Comment:
I honestly thought this was always here already. Not sure why it wasn't but we should definitely have this setup at the start of a suite run.